tools/common.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
+
+module.exports = {
+  checkMark: '✓',
```

Comment:
should we use non utf8 chars? e.g. 
```html
&#10003;
```

Reply:
Is it safe to assume that our sources are encoded in UTF8 charset? If so, why should we avoid using UTF8 chars?